DataDog/integrations-core
# NFSIOStat Check

### What does this PR do?

Adds a check for nfsiostat

### Motivation

Someone wanted one.

### Testing Guidelines

I have a very simple test to ensure that the metrics exist, its mocked and run as a default

### Additional Notes
Should we parse the data from `/proc/self/mountstats` ourselves?

We will need to merge [this PR](https://github.com/DataDog/omnibus-software/pull/162) and [this PR](https://github.com/DataDog/dd-agent-omnibus/pull/196) and release an agent as well. I have included a fallback in the check so that it will work without that as well.
